src/bot.js

```diff
@@ -87,6 +86,56 @@ let initializeBot = () => {
 				logger.info('tft command...');
 				message.channel.send(('https://tracker.gg/tft/profile/riot/NA/'+data+'/overview'));
 			}
+
+
+
+			if (command === 'store' && data) {		
+				logger.info('store command...');
+
+				const StoreCommand = splitMessage[2];
+				logger.info(StoreCommand);
+
+				//creates empty string to be added to
```

Comment:
take out the comments too - unless the code is unreadable, you can generally abstain from them in the codebase. (I know 3500 expects different xd)